packages/app/src/components/nominationInfo/EditCard.js

```diff
@@ -1,59 +1,117 @@
-import React, { useState, useEffect, useRef } from 'react';
+import React from 'react';
 import DatePicker from 'react-datepicker';
 import styles from './newstyles.module.css';
 import { Controller } from 'react-hook-form';
 import Title from './labelTypes/Title';
 import Other from './labelTypes/Other';
 import Provider from './labelTypes/Provider';
 import './calendar.css';
+import Select from 'react-select';
 
+const EditCard = (props) => {
+  // options for React-Select Dropdown
+  const options = [
+    { value: 'Yes', label: 'Yes' },
+    { value: 'No', label: 'No' },
+  ];
 
-const EditCard = props => {
-
-  // state for React Datepicker
-
-  const [admissionDate, setAdmissionDate] = useState(
-    new Date(props.formData['Admission Date'])
-  );
-
-  // state for React Datepicker
-  const [dischargeDate, setDischargeDate] = useState(
-    new Date(props.formData['Discharge Date'])
-  );
+  // styles for React-Select Dropdown
```

Comment:
all styles added for the React-Select - we are overriding the default styles it comes with

https://react-select.com/styles#styles